aws/application/application.template

```diff
@@ -440,6 +440,7 @@ Resources:
         Statement:
         - Effect: Allow
           Principal:
+            AWS: [arn:aws:iam::902837325998:role/LAA2ndLineRole]
```

Comment:
Is this correct if I want to allow people with the `LAA2ndLineRole` permission to assume the IAM Role attached to the EC2 Task? I'd like to do this so we can test locally.

Reply:
Worth a try.